recipes/cost733class/bld.bat

```diff
@@ -0,0 +1,31 @@
+
+set "cwd=%cd%"
+
+set "LIBRARY_PREFIX=%LIBRARY_PREFIX:\=/%"
+::set "MINGWBIN=%LIBRARY_PREFIX%/mingw-w64/bin"
+
+::  -D CMAKE_C_COMPILER:PATH=%MINGWBIN%/gcc.exe ^
+::  -D CMAKE_Fortran_COMPILER:PATH=%MINGWBIN%/gfortran.exe ^
+::  -D CMAKE_GNUtoMS:BOOL=ON ^
+
+set BUILD_TYPE=Release
+
+set "LDFLAGS=-L%LIBRARY_PREFIX%/lib -Wl,-rpath,%LIBRARY_PREFIX%/lib -lnetcdff"
+set "CFLAGS=-fPIC -I%LIBRARY_PREFIX%/include"
+
+mkdir build 
+cd build
+
+cmake -LAH -G "MinGW Makefiles" ^
+  %CMAKE_ARGS% ^
+  -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+  -D CMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^
+  -D CMAKE_BUILD_TYPE=%BUILD_TYPE% ^
+  -D GRIB=OFF ^
+  -D NCDF=ON ^
+  -D OPENGL=OFF ^
+  %SRC_DIR%
+if errorlevel 1 exit 1
+
+cmake --build . --target install --config %BUILD_TYPE%
```

Comment:
Is there a `make check` or `ctest` we can run? I'm not sure if I asked this already but I cannot find a comment on this.

Reply:
No, there is no check/ctest.

Just there is a test in the recipe checking that the binary has been created

Reply:
> Just there is a test in the recipe checking that the binary has been created

I wonder if there is anything we can to test it better than just that check?

Reply:
The original doc:
https://git.rz.uni-augsburg.de/philipan/cost733class-1.4/-/blob/main/doc/cost733class_userguide.pdf

has some examples with an sample data file (`slp.dat`) in the source repo, using different methods of classifications, but they are too large in CPU, WALLTIME and MEMORY, for a test. 

The original source repo, also has some example scripts, but for GUI output which is not suitable for test the build. 

If you don't mind, I would suggest just to left the current check and sort the recipe out.

Reply:
> but they are too large in CPU, WALLTIME and MEMORY, for a test.

How large? Do you mind testing it out first and, if it fails here due to lack of resources, we can leave the simpler test.